enclave-runner/src/loader.rs

```diff
@@ -200,6 +205,10 @@ impl<'a> EnclaveBuilder<'a> {
         self
     }
 
+    pub fn user_call_ext_impl(&mut self, e: Box<UsercallExtension>) {
```

Comment:
Call this `usercall_extension`. Change the parameter name to not be a single letter